SCYLLA-VERSION-GEN

```diff
@@ -69,13 +69,18 @@ then
 else
 	DATE=$(date --utc +%Y%m%d)
 	GIT_COMMIT=$(git -C "$SCRIPT_DIR" log --pretty=format:'%h' -n 1 --abbrev=12)
-	SCYLLA_VERSION=$VERSION
+	SCYLLA_VERSION=$(echo $VERSION | awk -F'-' '{print $1}')
+	RELEASE_CANDIDATE=$(echo $VERSION | awk -F'-' '{print $2}')
```

Comment:
Why pile more and more complexity into this thing.

Reply:
When we are building rpm packages the spec file can't have `-` as part of either `version` or `release` parameters. So what we need is to add the `dev` or in releases `rcX` as part of the `release` parameter
For ubuntu it's working

Reply:
rpm/deb define the version field as the software version, with release only about post-packaging changes. So pushing rc there is not correct.